### DISCUSSION TITLE
err in foundry.toml  ..... why red lines

### DISCUSSION THREAD

**Discussion by hammadshub** [Cyfrin/foundry-full-course-cu#984] [Q&A] [answered]
![Screenshot (40)](https://github.com/Cyfrin/foundry-full-course-f23/assets/112611922/2816bd39-12a3-4a9a-8860-6037883af5bb)

![Screenshot (41)](https://github.com/Cyfrin/foundry-full-course-f23/assets/112611922/83b96a37-03be-44f1-afd4-491e5b755f65)

Any Sol?

**Comment by theirrationalone** [comment]
Please install the `Better toml` extension on your `vs-code` wsl instance. If installed already Please try to, `enable` the `Better Toml` extension. Reach to that by clicking on extension icon available somewhere into left on vs-code. If you already have `enabled` it then Please try to restart your vs-code at least.

**Reply by hammadshub** [reply]
@theirrationalone 
I'm using "Even Better TOML" instead of "Better Toml".... Better toml has some issue in instlling.

I restarted but issue is not resolved?

**Reply by theirrationalone** [reply]
Yeah that's better though, Would you like to share your extension's opened window screenshots??

**Reply by hammadshub** [reply]
@theirrationalone 
![Screenshot (42)](https://github.com/Cyfrin/foundry-full-course-f23/assets/112611922/653d264d-98be-454a-8362-b97b362bc940)

**Reply by theirrationalone** [reply, marked as answer]
Heyy @hammadshub ! Sorry to say, But this extension requires lots of work to config our desired `TOML SYNTAX` highlighting. Please uninstall this extension why?? We want to focus on solidity & smart contracts development not on something our eyes like to see on screen.

A better option that i'll recommend you to is... [be5invis.toml](https://marketplace.visualstudio.com/items?itemName=be5invis.toml)

You can see its supremacy & simplicity.... 👇(Just One click and will back to our work) reload might a required action 😅.

![Screenshot (170)](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/ec20cb90-fcb4-409b-bd2e-63f54961957d)

;`)

**Reply by hammadshub** [reply]
It worked , thanks for helping.
